packages/@guardian/source-foundations/src/palette.ts

```diff
@@ -96,7 +96,7 @@ const colors = {
 		'#E4E5E8', //specialReport-700
 		'#EFF1F2', //specialReport-800
 	],
-};
+} as const;
```

Comment:
Is this needed?

Reply:
ah, I think this was a mistake, added a few weeks ago. will remove it!